app/controllers/events_controller.rb

```diff
@@ -1,3 +1,48 @@
 class EventsController < ApplicationController
-  def index; end
-end
+  def index
+    @events = Event.all
+  end
+
+  def show
+    @event = Event.all
```

Comment:
The `show` action should find a single event, not all events.

Reply:
Same here

Reply:
Thanks, resolved!